v2/datastream-to-spanner/src/test/resources/DataStreamToSpannerShardedMigrationIT/Movie-shard1.jsonl

```diff
@@ -0,0 +1,6 @@
+{"uuid":"f5dc7f9a-2bf5-4075-bbdd-844800000011","read_timestamp":"2024-02-07T06:44:40.209Z","source_timestamp":"2024-02-07T06:44:13.000Z","object":"S1L1_Movie","read_method":"mysql-cdc-binlog","stream_name":"projects/545418958905/locations/us-central1/streams/djagaluru-gen-data-json","schema_key":"a64df32fd8cf9187cb386a2900e15e307fd17923","sort_keys":[1707288253000,"mysql-bin.000013",6191653],"source_metadata":{"table":"Movie","database":"S1L1","primary_keys":["id1","id2"],"log_file":"mysql-bin.000013","log_position":6191653,"change_type":"UPDATE-INSERT","is_deleted":false},"payload":{"id1":1,"id2":1,"name":"Mov123","actor":27}}
```

Comment:
We should cover all data types in this test.

Reply:
All the data types will be covered in another test case.